install_config/aggregate_logging.adoc

```diff
@@ -780,11 +780,150 @@ ifdef::openshift-enterprise[]
 You can use `oc edit dc/logging-fluentd` to update your Fluentd configuration,
 making sure to first scale down your number of replicas to zero before editing
 the deployment configuration.
-enddef::openshift-enterprise[]
+endif::openshift-enterprise[]
 
 [NOTE]
 ====
 If you are not using the provided Kibana and Elasticsearch images, you will not
 have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
+
+[[configuring-curator]]
+== Configuring Curator
+
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+With Aggregated Logging version 3.2.1, Curator is available for use as Tech Preview.
+To start it, after completing an installation using the 3.2.1 Deployer, scale up
+the Curator deployment config that was created (defaults to zero replicas).
+====
+endif::openshift-enterprise[]
+
+Curator allows administrators to configure scheduled Elasticsearch maintenance operations
+to be performed automatically on a per-project basis. It is scheduled to run every
+day within its pod and will perform actions based on its configuration and recommended
+that only one pod be deployer per Elasticsearch cluster. Curator is configured
```

Comment:
Bit of a run-on sentence. How about:

It is scheduled to perform curation daily based on its configuration. Only one curator pod is recommended per Elasticsearch cluster. Curator is configured...